README.md

```diff
@@ -1,10 +1,48 @@
+
 # Valkyrien Warfare
 [![Build 
 Status](https://jenkins.daporkchop.net/job/Valkyrien-Warfare-Revamped/badge/icon)](https://jenkins.daporkchop.net/job/Valkyrien-Warfare-Revamped/)
+See: [LICENSE file](https://github.com/ValkyrienWarfare/Valkyrien-Warfare-Revamped/blob/8778e9d45f16c1f60e8149ab6cbdbabbdebc1278/LICENSE)
+
+The Airships Mod to end all other Airships Mods. Better compatibility, performance, collisions, interactions and physics than anything prior!
+
+
+## Installation
+
+### Downloading
+Official and stable releases of the Valkyrien Warfare mod can be found on the [CurseForge page](https://minecraft.curseforge.com/projects/valkyrien-warfare/files).
+
+Beta releases (warning: may be unstable!) can be found on the [Jenkins build server](https://jenkins.daporkchop.net/job/Valkyrien-Warfare-Revamped/).
+
+### Installing on your server
+To install Valkyrien Warfare, move the downloaded `.jar` file into your Minecraft's `mods/` folder, just as you would any other mod.
+
+## Usage
+
+### Make a flying ship
+1. Build a structure somewhere in the air, not connected to the ground.
+2. Place a [**Physics Infuser**] on it.
+3. Right click the [**Physics Infuser**] (the ship should fall).
+4. Place a [**Pilot Chair**] on the ship.
```

Comment:
pilot chair is very obsolete, and it's current mechanics are only for testing. would be better to only outline how to use the thrust modulator/ether compressor/engine system (maybe a wiki page would be better suited to this?)

Reply:
I think it's fine to give the pilot chair in the very basic tutorial, even just for testing, just so people can get that instant gratification of something just _working_, and then the more complicated systems can be explained in another section.

Reply:
i see what you mean, sure why not